advisor/src/main/kotlin/Advisor.kt

```diff
@@ -51,7 +51,7 @@ class Advisor(
     }
 
     @JvmOverloads
-    fun retrieveFindings(ortResult: OrtResult, skipExcluded: Boolean = false): OrtResult {
+    fun advise(ortResult: OrtResult, skipExcluded: Boolean = false): OrtResult {
```

Comment:
Commit message: "This align*s*..."

Reply:
While you are at it, would it make sense to make the `advise` function also a suspend function analogously to the API of the scanner? The advise providers already are suspendable.

Reply:
@sschuberth  Fixed, I also added some commitlint fixes.

Reply:
@mnonnenmacher, if you want, I could create a corresponding PR.

Reply:
I'd appreciate to see that, too, @oheger-bosch 😁